litestar/cli/_utils.py

```diff
@@ -271,7 +271,7 @@ def wrapped(ctx: Context, /, *args: P.args, **kwargs: P.kwargs) -> T:
 
         return func(*args, **kwargs)
 
-    return pass_context(wrapped)  # pyright: ignore
+    return pass_context(wrapped)  # type: ignore[arg-type]
```

Comment:
```
litestar/cli/_utils.py:274: error: Argument 1 to "pass_context" has incompatible type "Callable[[Context, **P], T]"; expected "Callable[[Context, Context, **P], T]"  [arg-type]
```
I ran out of time to debug this, but I'm wondering if a mypy bug given that pyright is OK with this now. Only guessing though.

Reply:
The issue was that the decorator `pass_context()` returns a callable that doesn't include `Context` as the first argument, but the `_inject_args()` func return annotation was `Callable[Concatenate[Context, P], T]`.